app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -571,12 +556,21 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
         }
         return dialog;
     }
-    
+
     @Override
     public boolean isBackEnabled() {
-        return false;
+        return true;
     }
-    
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();

```

Comment:
I don't think that's true? http://developer.android.com/reference/android/app/Activity.html#onBackPressed()